arduino-ide-extension/src/browser/widgets/sketchbook/sketchbook-tree-widget.tsx

```diff
@@ -134,26 +133,37 @@ export class SketchbookTreeWidget extends FileTreeWidget {
   protected renderInlineCommands(node: TreeNode): React.ReactNode {
     if (SketchbookTree.SketchDirNode.is(node) && node.commands) {
       return Array.from(new Set(node.commands)).map((command) =>
-        this.renderInlineCommand(command.id, node)
+        this.renderInlineCommand(
+          Array.isArray(command)
+            ? command
+            : typeof command === 'string'
+            ? command
+            : command.id,
+          node
+        )
       );
     }
     return undefined;
   }
 
   protected renderInlineCommand(
-    commandId: string,
+    command: string | [command: string, label: string],
     node: SketchbookTree.SketchDirNode,
     options?: any
   ): React.ReactNode {
-    const command = this.commandRegistry.getCommand(commandId);
-    const icon = command?.iconClass;
+    const commandId = Array.isArray(command) ? command[0] : command;
```

Comment:
Follow-up comment of the one I made in `renderInlineCommands`:
if you change the type of the `command` argument to accept a `Command | string | [command: string, label: string]`, you can move the check I mentioned here, like this:
```suggestion
    const commandId = Command.is(command)
      ? command.id
      : Array.isArray(command)
      ? command[0]
      : command;
```